components/profile/my-network-settings/governance-settings.tsx

```diff
@@ -89,6 +90,18 @@ export default function GovernanceSettings({
       });
   }
 
+  function handleNetworkTokens(tokens: Token[]) {
+    return tokens.map(token => ({
+        ...token,
+        isReward: token.network_tokens.isReward || false,
+        isTransactional: token.network_tokens.isTransactional || false
```

Comment:
Using double negation will result in `false` when `token.network_tokens.isReward` is `null`, so you can use `!!token.network_tokens.isReward` instead